src/server/wesnothd/server.cpp

```diff
@@ -211,7 +211,7 @@ const std::string help_msg =
 	" dul|deny_unregistered_login [yes|no], kick <mask> [<reason>],"
 	" k[ick]ban <mask> <time> <reason>, help, games, metrics,"
 	" netstats [all], [lobby]msg <message>, motd [<message>],"
-	" pm|privatemsg <nickname> <message>, requests, sample, searchlog <mask>,"
+	" pm|privatemsg <nickname> <message>, requests, roll, sample, searchlog <mask>,"
```

Comment:
I don't think the defaulting would be useful. It can confuse people who might run the command in hurry forgetting to put a number and getting a result in overlooked bounds.

The argument `<sides>` note ought to be set indeed.